### DISCUSSION TITLE
Conditional PR creation for git submodules

### DISCUSSION THREAD

**Discussion by gunjitchhhatwal** [renovatebot/renovate#31237] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### Please tell us more about your question or problem

I have a situation where I only need to update submodule commit version if a downstream step succeeds. E.g. - As of now every time a new change is pushed to the submodule repo's main branch, renovate creates a new PR. However, there are certain steps after the submodule repo is updated, such as successful execution of a pipeline which indicate that the commit can indeed be used by my repo (where the PR is created by renovate). 
Is there a way to achieve this?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
I don't think so

**Reply by gunjitchhhatwal** [reply]
Thanks for confirming.
